uis/layover/app/partials/hosted-edit.html

```diff
@@ -45,6 +45,11 @@
           <label>Readonly?</label>
           <span class="hint">If set to readonly, all uploading and deleting operations to this repo are prohibited</span>
         </div>
+        <div class="detail-field">
```

Comment:
Can we make it so marking a hosted repo as readonly will imply that it's also authoritative? I think that's always true, but it might cause the authoritative flag to be stuck == true if the user then goes back and flips the readonly flag back to false.

I wonder if the logic for authoritative (both in UI and in the content index handling logic) should allow readonly to imply authoritative indexing?